src/main/java/firebase/FirebaseInitializer.java

```diff
@@ -12,7 +12,7 @@ public class FirebaseInitializer {
 
     public static Firestore initializeFirebase() {
         try {
-            FileInputStream serviceAccount = new FileInputStream("/Users/masaarja/Desktop/geoguesser_firebase_service_account.json");
+            FileInputStream serviceAccount = new FileInputStream("/Users/puravgupta/IdeaProjects/geoguesser_firebase_service_account.json");
```

Comment:
can u not commit this cuz if we pull it we all have to change it back to whatever it is on our computers rn